_posts/francois_gerthoffert/2017-08-21-Infrastructure-procurement-in-large-scale-genomic-projects.md

```diff
@@ -0,0 +1,163 @@
+
+## Some numbers
+
+It we look back at our purchases, and focus only on disk and storage server costs (which only gives part of the picture), we get the following table:
+
+| Purchase date  | Disk size  | Cost per Disk  | Cost per disk-TB  | Cost per Server | Server Capacity | Cost per server-TB |
+|---|---|---|---|---|---|---|
+|May 2015       | 4 TB | $270   | $67 | $18,000 | 144 TB | $125 |
+|December 2015  | 6 TB | $380   | $63 | $24,000 | 216 TB | $111 |
+|September 2016 | 8 TB | $550   | $69 | $29,000 | 288 TB | $101 |
+|July 2017      | 10 TB| $620   | $62 | $32,000 | 360 TB | $89  |
+
+___Note: Above prices have been altered but represent a meaningful demonstration___
+
+Although there was a slight spike in September 2016 on individual disk cost, we can see that overall the cost per TB throughout our purchases went down.
+
+## Looking back
+
+Looking back is always easy, but back when we established our procurement strategy, we could only rely on our past experience in the industry, with no guarantees that our strategy relying on purchasing "as late as possible" was going to be effective.
+
+One of the external factors is the USD -> CDN conversion rate, currency fluctuations directly influence our purchase prices. It probably explains why the cost per TB on an individual disk was higher in September 2016. Our storage need at that time was only of 3 servers (864 TB), so the impact was fairly limited and the conversion rate started to go back down after that time and is today at the same level it was early 2015 (although still much higher than when the project started in 2014).
+
+<center>
+  <figure style="width: 70%;">
+      <img src="{{site.urlimg}}francois_gerthoffert/collab-procurement/usd-cdn-conversion-rate.png"/>
+      <figcaption>USD-CDN exchange rate evolution</figcaption>
+  </figure>
+</center>
+
+But ultimately, the thing to look at is that today we are purchasing servers with 2.5 times more storage capacity than in 2015, which means a need for less physical resources per TB of storage, which plays largely in our favor from a budget point of view.
+
+# Conclusion
+
+So what could be the recommendations and conclusion of this article ?
+
+If you know upfront that your project will be composed of multiple purchases, identify a procurement strategy and setup a contractual framework before starting. Not having to re-negotiate or re-validate items such as warranty, support details, greatly eases the procurement process.
+
+Don't purchase more storage capacity than required. Considering that transferring petabytes of data takes a large amount of time, you might have sufficient time for staggered purchases. Instead, identify your requirements and schedule your purchases accordingly. This will provide the project with better value per dollar spent.
```

Comment:
Don't purchase more storage capacity than required. Considering that transferring petabytes of data takes a long time, and the data to be stored might not yet be generated or ready to be stored, you might have sufficient time for staggered purchases. Instead, identify your requirements and schedule your purchases accordingly. This will provide the project with better value per dollar spent by benefitting of cost decreases and technological improvements.